src/routes/news/[...slug].svelte

```diff
@@ -0,0 +1,152 @@
+<script context="module">
+    import newsTargets from 'news-targets';
+
+    export async function preload({ params, query }) {
+        // the `slug` parameter is available because
+        // this file is called [slug].svelte
+        const [slug, title] = params.slug;
```

Comment:
Why do we have slug and title in the news url? I believed slug is meant to be unique.